lib/pushmi_pullyu/preservation_queue.rb

```diff
@@ -61,11 +61,11 @@ def next_item
   end
 
   def wait_next_item
-    # loop do
-    element = next_item
-    return element if element.present?
-    sleep @poll_interval
-    # end
+    while PushmiPullyu.continue_run_loop?
```

Comment:
I know this name was my idea, but seeing it now maybe `continue_run_loop?` is the wrong name, considering that we don't use it in the main run loop and it can return false while we still want to continue the main run loop (when we need to rotate the logs).

`continue_polling?` maybe?